.github/workflows/build_and_deploy.yml

```diff
@@ -8,12 +8,14 @@ on:
     branches:
      - master
  workflow_dispatch:
+env:
+  ORG_NAME: BahmniIndiaDistro
+  EVENT_TYPE: hip-module-trigger
 
 jobs:
   build:
-
+    name: build and deploy hip omod
```

Comment:
This job is actually not deploying, we can rename it to build and generate artefact

Reply:
we can rename the file as well